pkg/network/usm/ebpf_main.go

```diff
@@ -55,9 +52,11 @@ type ebpfProgram struct {
 	cfg                   *config.Config
 	subprograms           []subprogram
 	probesResolvers       []probeResolver
-	mapCleaner            *ddebpf.MapCleaner
 	tailCallRouter        []manager.TailCallRoute
 	connectionProtocolMap *ebpf.Map
+
+	protocols         map[protocols.ProtocolType]protocols.Protocol
+	disabledProtocols []*protocols.ProtocolSpec
```

Comment:
seeing those 2 fields all together make me wonder if `protocols` represents all protocols, or just enabled protocols

Reply:
I agree. Renamed to `enabledProtocols`.